src/python/WMComponent/AgentStatusWatcher/DrainStatusPoller.py

```diff
@@ -73,6 +78,28 @@ def getDrainInfo(cls):
         """
         return cls.drainStats
 
+    def checkDrainStatusChanges(self, previousConfig):
+        """
+        Check to see if any drain statuses have changed in the auxiliary db
+        If yes, send email notification and update local drain thread variables
+
+        """
+        message = ""
+        drainStatusKeys = ['UserDrainMode', 'AgentDrainMode', 'SpeedDrainMode']
+
+        if not previousConfig:
+            return
+
+        for key in drainStatusKeys:
+            if previousConfig[key] != self.agentConfig[key]:
+                message += "Agent had a drain status transition. %s = %s\n" % (str(key), str(self.agentConfig[key]))
```

Comment:
How about replacing that dot by "to"? So
`Agent had a drain status transition to: `...

Reply:
BTW, where is the host name visible? Maybe in the sender? If not, then we should update it to actually:
`agent %s had a drain status transition to: % host`
replace `host` by the var hosting the agent name :)

Reply:
The agent hostname is in the email subject